feat(docs) adding new blog post for Big Data LDN (#1136)
questdb/questdb.io: blog/2022-10-20-questdb-big-data-ldn.md

---
title: "QuestDB at Big Data LDN 2022"
author: Javier Ramirez
author_title: QuestDB Team
author_url: https://github.com/javier
author_image_url: https://avatars.githubusercontent.com/u/3839?v=4
description:
  Big Data LDN (London) is the UK’s leading free to attend data & analytics
  conference and exhibition. This year, Javier Ramirez, Developer Advocate at
  QuestDB, delivered a talk on "Ingesting A Million Time Series Per Second On A
  Single Instance".
keywords: [bigdata, questdb, timeseries, event]
tags: [community, bigdata, questdb, timeseries, event]
image: /img/blog/2022-10-20/big_data.png
---

import Screenshot from "@theme/Screenshot"

> Big Data LDN (London) is the UK’s leading free to attend data & analytics

Reviewer: Big Data LDN (London) is the UK leading data & analytics

> Big Data LDN (London) is the UK’s leading free to attend data & analytics

Reviewer 2: ```suggestion
Big Data LDN (London) is the UK leading data & analytics
```

conference and exhibition. This year,
[Javier Ramirez](https://github.com/javier), Developer Advocate at QuestDB,
delivered a talk on "Ingesting A Million Time Series Per Second On A Single

> Instance". Well done, Javier!

Reviewer: The author is Javier, but here the writer is somebody else than Javier.

<!--truncate-->

> Big Data LDN was very likely the largest event focusing on data in the UK, with

Reviewer: Big Data LDN is the largest event focusing on data in the UK,

> Big Data LDN was very likely the largest event focusing on data in the UK, with

Reviewer 2: ```suggestion
Big Data LDN is the largest event focusing on data in the UK,
```

two days of talks in parallel tracks, plus dozens of vendors. This was a great

> opportunity to see trending topics and talk with data-minded folks. We asked

Reviewer: opportunity to learn about the latest trends and engage with data-minded folks.

> opportunity to see trending topics and talk with data-minded folks. We asked

Reviewer 2: ```suggestion
opportunity to learn about the latest trends and engage with data-minded folks.
```

A good mix of data engineers, data analysts, decision makers, data vendors, and
some students.

### Who did you meet?

We actually met a few QuestDB users, which was an absolute pleasure. We also met
a lot of companies with interesting near real time challenges looking for
solutions, and we found some time to talk to other vendors, explore
collaborations, or simply have a friendly talk about our products. Users were
from many different backgrounds: from companies managing wireless networks
producing thousands of events per millisecond, to cryptocurrency exchanges, and
we even had a great time speaking with a F1 team looking for a time series
database. We hope to see them all soon in our community slack channel!

### What did you talk about?

Mostly about fast and big data, but also quite a bit about the internals of
QuestDB and what makes us stand out from the rest. In particular, our latest
developments on optimizing imports using `io_uring`, or the fact that we use JAVA
with near zero Garbage Collection were popular topics. But we mostly talked
about real use cases and constraints, and how QuestDB could help.

### What are some memorable questions people asked you?

> - Are you built from scratch or on top of some other engine?

Reviewer 3: ```suggestion
- Is QuestDB built from scratch or on top of some other engine?
```

- How are you going to monetize an open source database?

> - Are you really only 20 people?

Author: I'd leave this as the answer is in the question and it is more surprising?
